app/views/pages/index.html.erb

```diff
@@ -0,0 +1,35 @@
+<% set_page_title(@form.name) %>
+
+<% content_for :back_link, govuk_back_link_to(form_path(@form), "Back to create a form") %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <span class="govuk-caption-l"><%= @form.name %></span>
+    <h1 class="govuk-heading-l"><%= t("pages.index.title") %></h1>
+
+    <% if flash[:message] %>
+      <p class="govuk-body"><%= flash[:message] %></p>
+    <% end %>
+
+    <%= link_to t("pages.index.add_question"), new_page_path(@form), class: "govuk-button govuk-!-margin-bottom-3 govuk-!-margin-top-3", 'data-module':"govuk-button", draggable:"false", role: "button" %>
+
+    <% if @pages.any? %>
+      <h2 class="govuk-heading-m govuk-!-margin-top-7 govuk-!-margin-bottom-0"><%= t("forms.form_overview.your_questions") %></h2>
+      <% @pages.each do |page| %>
+        <dl class="govuk-summary-list">
+          <div class="govuk-summary-list__row">
+            <dt class="govuk-summary-list__key govuk-!-font-weight-regular">
+              <%= page.question_text %>
+            </dt>
+            <dd class="govuk-summary-list__actions">
+              <%= link_to edit_page_path(@form,page), class: "govuk-link govuk-link--no-visited-state" do %>
```

Comment:
Similar to above - think this could be a `govuk_link_to`:
```suggestion
              <%= govuk_link_to edit_page_path(@form,page), no_visited_state: true do %>
```

Reply:
🤔 why do we want to turn off the visited state, is that not an accessibility issue if thats done because users might not realise that they have already visited that page before

Reply:
Bit of a grey area - it's based on this [design system guidance on links](https://design-system.service.gov.uk/styles/typography/#links):

> Use the govuk-link--no-visited-state modifier class where it is not helpful to distinguish between visited and unvisited states, for example when linking to pages with frequently-changing content such as the dashboard for an admin interface.

I think this is in there because it's content that will potentially change pretty frequently as a form goes through review processes etc, so the state a user has previously seen it in won't necessarily be the current state. But I think that you're probably right that we should get rid of the `no_visited_state: true` - if we find that users are getting conused, we can change it back, but it makes sense to keep the visited state by default.

Reply:
Design/Content [decided](https://gds.slack.com/archives/C030K5706TX/p1661155640382409) we should keep the default link behaviour.